src/graphql/account.rs

```diff
@@ -301,7 +301,15 @@ impl AccountMutation {
         Ok(username)
     }
 
-    /// Authenticates with the given username and password
+    /// Authenticates with the given username and password.
+    ///
+    /// If the `lastSigninTime` value of the `account` is `None`, the operation will fail, and
+    /// it should be guided to call `signInWithNewPassword` GraphQL API.
+    ///
+    /// # Errors
```

Comment:
It would be better to insert an empty line between the title and the content.

Reply:
Thank you for checking the mistake.